SRP/Example5.md

```diff
@@ -0,0 +1,24 @@
+### Usage
+
+---
+#### Where
+
+https://github.com/angular/angular/blob/main/packages/animations/src/animation_builder.ts#L57
```

Comment:
➕
https://stackoverflow.com/questions/46790893/abstract-factory-violate-single-responsibility-principle#:~:text=The%20Abstract%20Factory%20has%20a,responsibility%20not%20making%20the%20objects.